src/sagas/wallet.js

```diff
@@ -86,7 +86,7 @@ export function* startWallet(action) {
 
   // If we've lost redux data, we could not properly stop the wallet object
   // then we don't know if we've cleaned up the wallet data in the storage
-  walletUtil.cleanLoadedData();
+  walletUtil.cleanLoadedData({ cleanAccessData: false });
```

Comment:
Maybe explain why we can't clean access data here?

Mainly because this is now different from the desktop wallet, where we have:

```
    oldWalletUtil.cleanLoadedData({ cleanAccessData: !fromXpriv});
```

But we clean loaded data only if it's not wallet service. It's important to understand why they are different.

Reply:
Improved the comment here, explaining why I am not cleaning the accessData anymore

ce09e21c0bbf54758cb93881d8f23c71c60a7205